ktor-client/ktor-client-android/jvm/src/io/ktor/client/engine/android/AndroidClientEngine.kt

```diff
@@ -115,3 +120,6 @@ internal fun HttpURLConnection.content(callScope: CoroutineContext): ByteReadCha
 } catch (_: IOException) {
     errorStream?.buffered()
 }?.toByteReadChannel(context = callScope, pool = KtorDefaultPool) ?: ByteReadChannel.Empty
+
+@Suppress("KDocMissingDocumentation")
+class RequestInvalidException(override val message: String): IllegalStateException()
```

Comment:
Do we need it public? Shouldn't it be internal instead? Or shared between engines?